docs/section5/Rule5-29_nonprimary.md

```diff
@@ -0,0 +1,38 @@
+# Envelope - Rule 5-29  
+**Schema Version** 0.0.23  
+**Primary Rule:** True
+**Rule ID:** 5-29  
+**Rule Description:** Baseline fenestration shall be assumed to be flush with the exterior wall, and no shading projections shall be modeled.
```

Comment:
The language in table G3.1 is for "all vertical fenestration". Please add "vertical" to the rule description here and in the master spreadsheet.